app/src/main/java/com/sdp13epfl2021/projmag/model/Gender.kt

```diff
@@ -0,0 +1,17 @@
+package com.sdp13epfl2021.projmag.model
+
+enum class Gender {
+    MALE{
+        override fun toString(): String {
+            return "Male"
+        }
+        },FEMALE{
+        override fun toString(): String {
+            return "Female"
+        }
+                },OTHER{
+        override fun toString(): String {
+            return "Other"
+        }
+                }
+}
```

Comment:
This could be simplified with something like this
enum class Gender(private val text: String) {
    MALE("Male"), FEMALE("Female"), OTHER("Other");
    override fun toString(): String { return text }
}